src/utils/db.js

```diff
@@ -80,6 +81,11 @@ const iteratorToList = async function iteratorToList(iterator) {
                 logger.debug(err);
                 jsonRes.record = res.value.value.toString('utf8');
             }
+
+            if (returnTimestamps && res.value.timestamp) {
+                jsonRes.timestamp = new Date(res.value.timestamp.seconds.low * 1000).toISOString();
```

Comment:
why convert this to a string? Isn't it better to just keep the Date as part of the API ... else if you need the date for comparison or whatever it will become difficult maybe for getting back to a EPOCH format ...